agent/firewall.go

```diff
@@ -113,23 +117,26 @@ func (fw *Firewall) Init(netif NetIf) error {
 	// Allow ICMP, DHCP and SSH between host and instances.
 	hostAddr := fw.Agent.networkConfig.romanaGW
 	inputRules := []string{
-		fmt.Sprintf("-d %s/32 -p icmp -m icmp --icmp-type 0 -m state --state RELATED,ESTABLISHED", hostAddr),
-		fmt.Sprintf("-d %s/32 -p tcp -m tcp --sport 22", hostAddr),
 		"-d 255.255.255.255/32 -p udp -m udp --sport 68 --dport 67",
```

Comment:
That's correct. I'm trying to keep platform related changes code to a minimum in the agent.